src/estimagic/inference/bootstrap.py

```diff
@@ -17,9 +18,10 @@
 
 
 def bootstrap(
-    data,
     outcome,
     *,
+    data=None,
+    existing_outcomes=None,
```

Comment:
I think we have some notebooks in the documentation that use `bootstrap_from_outcomes`. @segsell can you check and adjust them? You should be most familiar because you just did the adjustment to the new results object.